pom.xml

```diff
@@ -405,6 +405,13 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.scalatestplus</groupId>
+                <artifactId>scalatestplus-scalacheck_${scala.binary.version}</artifactId>
+                <version>3.1.0.0-RC2</version>
```

Comment:
cc @pan3793

Reply:
Can we switch to 
```
<dependency>
    <groupId>org.scalatestplus</groupId>
    <artifactId>scalacheck-1-15_2.12</artifactId>
    <version>3.2.9.0</version>
    <scope>test</scope>
</dependency>
```
?

Reply:
BTW, please always extract `version` as property.

Reply:
Updated.